src/semeio/fmudesign/create_design.py

```diff
@@ -17,6 +17,36 @@
 from semeio.fmudesign.iman_conover import ImanConover
 
 
+def is_consistent_correlation_matrix(matrix):
+    # Convert to numpy array if not already
+    matrix = np.array(matrix)
+
+    # Check if matrix is 2D and square
+    if len(matrix.shape) != 2 or matrix.shape[0] != matrix.shape[1]:
+        raise ValueError("Correlation matrix must be 2D and square")
+
+    # Check symmetry
+    if not np.allclose(matrix, matrix.T):
+        raise ValueError("Correlation matrix must be symmetric")
+
+    # Check diagonal elements are 1
+    if not np.allclose(np.diagonal(matrix), 1):
+        raise ValueError("Correlation matrix diagonal elements must be 1")
+
+    # Check elements are between -1 and 1
+    if np.any(matrix > 1 + 1e-8) or np.any(matrix < -1 - 1e-8):
```

Comment:
perhaps these errors should be propagated out to print with a more detailed description? 
```
element C[row=1, column=3] = 1.001, but every correlation should be in range [0, 1]
```

Reply:
I removed this test as I think the definition of a consisten correlation matrix is that it's positive semi-definite and has ones on the diagonal. This is a different type of check I think. Might still be useful though. What do you think?